NitroxClient/Communication/Packets/Processors/ChatMessageProcessor.cs

```diff
@@ -19,14 +19,20 @@ public ChatMessageProcessor(PlayerManager remotePlayerManager, PlayerChatManager
 
         public override void Process(ChatMessage message)
         {
-            Optional<RemotePlayer> remotePlayer = remotePlayerManager.Find(message.PlayerId);
-
-            if (remotePlayer.IsPresent())
+            if (message.PlayerId != ushort.MaxValue)
             {
-                RemotePlayer remotePlayerInstance = remotePlayer.Get();
-                ChatLogEntry chatLogEntry = new ChatLogEntry(remotePlayerInstance.PlayerName, message.Text, remotePlayerInstance.PlayerSettings.PlayerColor);
-                chatManager.WriteChatLogEntry(chatLogEntry);
+                Optional<RemotePlayer> remotePlayer = remotePlayerManager.Find(message.PlayerId);
+
+                if (remotePlayer.IsPresent())
+                {
+                    RemotePlayer remotePlayerInstance = remotePlayer.Get();
+                    ChatLogEntry chatLogEntry = new ChatLogEntry(remotePlayerInstance.PlayerName, message.Text, remotePlayerInstance.PlayerSettings.PlayerColor);
+                    chatManager.WriteChatLogEntry(chatLogEntry);
+                }
+                return;
```

Comment:
Instead of a return, I suggest a small refactor to make more readable:

```
bool serverCommandMessage = (message.playerId == ChatMessage.SERVER_COMMAND_PLAYER_ID);

if(serverCommandMessage) {
    outputServerMessage(chatMessage);
} else {
    outputPlayerMessage(chatMessage);
}


```